src/dysh/fits/gbtfitsload.py

```diff
@@ -1847,11 +1847,23 @@ def _update_radesys(self):
 
         # Azimuth and elevation case.
         azel_mask = (self["CTYPE2"] == "AZ") & (self["CTYPE3"] == "EL")
+        # Update self._index.
         self._index.loc[azel_mask, "RADESYS"] = radesys["AzEl"]
+        # Update SDFITSLoad.index.
+        sdf_idx = set(self["FITSINDEX"][azel_mask])
+        for i in sdf_idx:
+            sdfi = self._sdf[i].index()
```

Comment:
`sdfi = self._sdf[i]`
should ould also work since SDFITSLoad has __getitem__

Reply:
In this case the update would require a chained indexing, which will be deprecated by `pandas`.